src/apis/avm/api.ts

```diff
@@ -1254,20 +1254,33 @@ export class AVMAPI extends JRPCAPI {
    * @param amount The amount of the asset to be sent
    * @param to The address of the recipient
    * @param from An array of addresses managed by the node's keystore for this blockchain which will fund this transaction
+   * @param changeAddr An address to send the change
+   * @param memo Optional CB58 Buffer or String which contains arbitrary bytes, up to 256 bytes
    *
    * @returns Promise for the string representing the transaction's ID.
    */
-  send = async (username:string, password:string, assetID:string | Buffer, amount:number | BN, to:string, from:Array<string> | Array<Buffer>):Promise<string> => {
+  send = async (username:string, password:string, assetID:string | Buffer, amount:number | BN, to:string, from:Array<string> | Array<Buffer>, changeAddr:string, memo:string | Buffer = undefined):Promise<{txID: string, changeAddr: string}> => {
     let asset:string;
     let amnt:BN;
+    let m:string;
 
     if (typeof this.parseAddress(to) === 'undefined') {
       /* istanbul ignore next */
       throw new Error(`Error - AVMAPI.send: Invalid address format ${to}`);
     }
 
+    if (typeof this.parseAddress(changeAddr) === 'undefined') {
```

Comment:
if changeAddress is optional, what is the behavior of parseAddress() when undefined is passed in?